packages/react-native-codegen/src/parsers/__tests__/parser-primitive-test.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Thanks for adding the test!
Could you please name this `parser-primitives-test.js`, for consistency?

Reply:
@rshest thanks for the review!
I realized that the original file was called `parsers-primitives.js`, both word containing `s` 
So I renamed the test to also follow the same `parsers-primitives-test.js`